layouts/partials/helpers/text-color.html

```diff
@@ -0,0 +1,13 @@
+{{- $bg := .self.Scratch.Get "bg" }}
+{{- $text_muted := cond (.muted | default false) "text-muted " "" -}}
```

Comment:
Muted does mostly make sense on light backgrounds. Assuming we have to codify the different variations anyway, I think it's easier to remove muted. What do you think?